sound/soc/sof/intel/hda-loader.c

```diff
@@ -78,26 +78,13 @@ static int cl_stream_prepare(struct snd_sof_dev *sdev, unsigned int format,
  * reset/stall and then turn it off
  */
 static int cl_dsp_init(struct snd_sof_dev *sdev, const void *fwdata,
-		       u32 fwsize)
+		       u32 fwsize, int stream_tag)
 {
-	int tag, ret, i;
+	int ret, i;
 	u32 hipcie;
-	struct sof_intel_hda_dev *hda =
-		(struct sof_intel_hda_dev *)sdev->pdata->hw_pdata;
+	struct sof_intel_hda_dev *hda = sdev->pdata->hw_pdata;
```

Comment:
reverse xmas tree please, we will get that feedback when upstreaming

Reply:
Sure, let me do it as already at here.